test/e2e-go/cli/goal/expect/goalAppAccountAddressTest.exp

```diff
@@ -112,19 +112,19 @@ proc goalAppAccountAddress { TEST_ALGO_DIR TEST_DATA_DIR} {
     ::AlgorandGoal::CheckProcessReturnedCode 1
 
     puts "Checking the results"
-    set EXPECTED_OUTPUT "Account0*$PRIMARY_ACCOUNT_ADDRESS"
-    set EXPECTED_OUTPUT "$EXPECTED_OUTPUT*Account1*$ACCOUNT_1_ADDRESS"
-    set EXPECTED_OUTPUT "$EXPECTED_OUTPUT*Account2*$ACCOUNT_2_ADDRESS"
-    set EXPECTED_OUTPUT "$EXPECTED_OUTPUT*Account3*$ACCOUNT_3_ADDRESS"
-    set EXPECTED_OUTPUT "$EXPECTED_OUTPUT*Account4*$ACCOUNT_4_ADDRESS"
-    set EXPECTED_OUTPUT "$EXPECTED_OUTPUT*Num Accounts*ui*4"
+    set EXPECTED_OUTPUT "Account0.*$PRIMARY_ACCOUNT_ADDRESS"
```

Comment:
The placeholder for `any once character` is `?`. `.` will match exactly `.`
Substituting`.*` with `?*` will fix the failure, and will evaluate one or more characters as expected.